test/compilable/test7172.d

```diff
@@ -1,3 +1,10 @@
+/*
+TEST_OUTPUT:
+---
+compilable/test7172.d(16): Deprecation: `scope` as a type constraint is deprecated.  Use `scope` at the usage site.
```

Comment:
You can just remove the class, that's not what the issue was about: https://issues.dlang.org/show_bug.cgi?id=7172